File: 1069.py
seq = input()

def repetitions(s):
    prev, res, curr = '!', 1, 1
    for c in s:
        if c != prev:
            prev, curr = c, 0
        curr += 1
        res = max(res, curr)
    return res

print(repetitions(seq))

File: 1069.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string s;
	cin >> s;
	char prev = '!';
	int cnt = 0, res = 0;

	for (char c : s) {
		cnt = c == prev ? cnt + 1 : 1;
		prev = c;
		res = max(res, cnt);
	}

	cout << res;
}
